.github/workflows/build.yml

```diff
@@ -111,28 +116,74 @@ jobs:
           echo "package direct-sqlcipher" >> cabal.project.local
           echo "    flags: +openssl"      >> cabal.project.local
 
-      - name: Unix build
-        id: unix_build
+      - name: Unix build CLI
+        id: unix_cli_build
         if: matrix.os != 'windows-latest'
         shell: bash
         run: |
           cabal build --enable-tests
           echo "::set-output name=bin_path::$(cabal list-bin simplex-chat)"
 
+      - name: Unix test
```

Comment:
better to keep this step after upload to release steps - it was moved as a workaround for sporadic test failures in CI

Reply:
ok